### PR TITLE
Cv classification integration

### PR DESCRIPTION
**Pull request by shijiew555**
Changes
Made file `classification.cpp` and completed skeleton class `Classification`. Color classification has not been implemented. Made file `cv_classification.cpp` for the integration test. 

Testing
To make the target for the integration test, use `make cv_classification`. Target images for integration test is in `tests/integration/images/` and image paths could be specified in the variable `targetPaths` in `cv_classification.cpp`. The paths for model weights files for segmentation, shape classification, and character classification can be specified in variables `segmentationModelPath`, ` classificationShapeModelPath`, and `classificationCharModelPath` respectively.

### CODE REVIEW

File: CMakeLists.txt
Comment (atar13) [inline CMakeLists.txt:166]: Can this be moved to `tests/integration/CMakeLists.txt`? I think your branch is out of sync with main since we recently moved all the integration tests there. But it's all good. You'll just have to merge the updated main into this branch and move this CMake snippet there.

File: include/cv/classification.hpp
Comment (atar13) [inline include/cv/classification.hpp:44]: I think you can remove the color Model private member here for now. Technically it's not a Pytorch model so we don't need to declare it with that type.

File: src/cv/classification.cpp
Comment (atar13) [inline src/cv/classification.cpp:8]: Thank you for adding the CUDA loading :1st_place_medal:

File: tests/integration/cv_classification.cpp
Comment (atar13) [inline tests/integration/cv_classification.cpp:49]: Really like this integration test. It's nice how it runs Segmentation first since that makes it easy for us to pass in generated image from not-stolen or real images to test.

**Review by atar13 — APPROVED** [review]
These changes look great and the code is well organized and well documented. Thanks for taking this project on and getting the last stage of the pipeline done!

We can worry about the color model in another PR if you just want to get this merged in now. Since the color model isn't a Pytorch model, it works pretty differently than all the other models so we can talk more about it later if you want.

The one blocker before merging this in is to merge in the updated main branch into this branch. If you want, I can take care of it since I'm more familiar with the rest of the codebase. Otherwise, feel free to lmk if you need any help with the merge.

**Review by Samir-Rashid — APPROVED** [review]
You can merge this if you are ready. Anthony covered everything with his feedback.

**Pull request merged**
